app/services/file_manager/file_upload/file_upload.py

```diff
@@ -119,6 +119,57 @@ def assemble_path(
     return current_folder_node, parent_folder, create_folder_flag, target_folder
 
 
+def item_duplication_check(
+    create_folder_flag: bool, file_objects: List[FileObject], upload_client: UploadClient
+) -> List[FileObject]:
+    '''
+    Summary:
+        The function will check if the file is already uploaded on the platform.
+        If the file is already uploaded, the function will ask if to skip the file.
+        Otherwise, the function will exit
+    Parameter:
+        - create_folder_flag(bool): the flag to indicate if need to create new folder
+        - file_objects(List[FileObject]): the list of file object
+        - upload_client(UploadClient): the upload client object
+    Return:
+        - non_duplicate_file_objects(List[FileObject]): the list of file object that is not duplicated
+    '''
+
+    # make the file duplication check to allow folde merging
+    non_duplicate_file_objects = []
```

Comment:
Should we include logging that indicates we are starting the item duplication check?